main.tf

```diff
@@ -123,13 +123,33 @@ resource "aws_s3_bucket" "default" {
     },
   )
 
+  logging {
+    target_bucket = aws_s3_bucket.default.id
+    target_prefix = "log/"
+  }
+
+  replication_configuration {
+    role = aws_iam_role.replication.arn
+    rules {
+      id     = "rule1"
+      prefix = ""
+      status = "Enabled"
+
+      destination {
+        bucket        = aws_s3_bucket.destination.arn
```

Comment:
I don't see a definition for a TF bucket resource called "destination" anywhere. You may need to define one.

AFAIK there is no pattern for cross-region replicated buckets, but there is a spike to define a viable solution (https://github.com/ministryofjustice/modernisation-platform/issues/1054).

Reply:
I'm not sure how big of a piece of work this will be. Feel free to leave a note on the ticket to make sure this bucket is covered by changes, if they are to be done at a later time.